src/main/java/org/prebid/server/analytics/reporter/agma/AgmaAnalyticsReporter.java

```diff
@@ -200,7 +206,9 @@ private static String getPublisherId(BidRequest bidRequest) {
             return null;
         }
 
-        return publisherId;
+        return appSiteId != null
+                ? String.format("%s_%s", StringUtils.defaultString(publisherId, ""), appSiteId)
```

Comment:
the second param is redundant, use just `StringUtils.defaultString(publisherId)`

also if it's a blank publisherId it will return `_appSiteId` value, imho just looks weird, but maybe it's fine for you